Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,721 @@
+                        'Description': vaccine['description'],
+                        "IsMonitorOnly": vaccine['isMonitorOnly'],
+                        'Last modified by': vaccine['lastModifiedBy'],
+                        'Last modified on': vaccine['lastModifiedOn']
+                    }
+                    context.append(vaccine_context)
+                ec['Minerva.Vaccine'] = context
+            else:
+                context = {
+                    'Id': data['id'],
+                    'Name': data['name'],
+                    'Type': data['type'],
+                    'Description': data['description'],
+                    "IsMonitorOnly": data['isMonitorOnly'],
+                    'Last modified by': data['lastModifiedBy'],
+                    'Last modified on': data['lastModifiedOn']
+                    }
+                ec['Minerva.Vaccine(val.Id === obj.Id)'] = context
+        elif context_id.endswith('exclusions'):
+            context = []
+            if isinstance(data, list):
+                for exclusion in data:
+                    exclusion_context = {
+                        'Id': exclusion['id'],
+                        'Type': exclusion['type'],
+                        'Exclusion data': exclusion['data'],
+                        'Description': exclusion['description'],
+                        'Last modified by': exclusion['lastModifiedBy'],
+                        'Last modified on': exclusion['lastModifiedOn'],
+                        'Applied groups': exclusion['appliedGroupsIds']
+                    }
+                    context.append(exclusion_context)
+                ec['Minerva.Exclusion'] = context
+            else:
+                context = {
+                    'Id': data['id'],
+                    'Type': data['type'],
+                    'Exclusion data': data['data'],
+                    'Description': data['description'],
+                    'Last modified by': data['lastModifiedBy'],
+                    'Last modified on': data['lastModifiedOn'],
+                    'Applied groups': data['appliedGroupsIds']
+                }
+                ec['Minerva.Exclusion(val.Id === obj.Id)'] = context
+        else:
+            LOG('Failed to create entry context for {}'.format(context_id))
+
+        return ec
+
+
+    ''' FUNCTIONS '''
+    def get_from_url(url):
+        url_text = url.split('/')[-1]
+        response = session.get(url)
+        if response.status_code is not 200:
+            return_error('Error while fetching {}. More information: {}, {}'.format(url_text, response.status_code, response.reason))
+
+        results = {
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': response.json(),
+            'HumanReadable': tableToMarkdown(pascalToSpace(url_text), response.json(), headerTransform=pascalToSpace, removeNull=True)
+        }
+
+        if response.json():
```

Comment:
I think that should be checked before `results` is created, and then return the results accordingly